### ISSUE TITLE
Blendrr crashes when importing from cc4

### ISSUE THREAD

**Issue by kissmecomix** [soupday/cc_blender_tools#278]
When importing a character from  CC4 using the CC4/IC* pipeline addon , NOT the data link,  as the character is importing, blender crashes. the character comes in fine when using the data link,. however, i have run into an issue with a mesh when it imports into blender and i wanted to see if it was the data link, or the mesh, thus i wsanted to use a different method to get the character in from cc$

**Comment by soupday** [comment]
Crashes on import are usually down to CC4 miss-categorizing the cloth meshes as hair or beards.

When it shows you the pop-up on import which lists each mesh and it's type: Make sure they are all correct.

**Comment by kissmecomix** [comment]
Now Blender is crashing even with datalink, now what do i do??

**Comment by soupday** [comment]
Yes it matters, iClone / CC doesn't isolate the code between versions, with multiple versions who knows which python scripts it's really using. Remove them all but the latest.
